esmvaltool/utils/color_tables/show_color_tables.py

```diff
@@ -0,0 +1,216 @@
+"""Utility script for inspecting and converting ncl color tables."""
+import argparse
+import glob
+import logging
+import os
+import warnings
+import yaml
+
+import matplotlib
+matplotlib.use("Agg")
+import matplotlib.pyplot as plt
+import numpy as np
+from jinja2 import Template
+
+from esmvaltool.diag_scripts.shared.plot import __file__ as plot_path
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.DEBUG)
+CONSOLE_HANDLER = logging.StreamHandler()
+CONSOLE_HANDLER.setLevel(logging.INFO)
+CONSOLE_HANDLER.setFormatter(
+    logging.Formatter('%(asctime)s - %(name)s - %(levelname)s - %(message)s'))
+logger.addHandler(CONSOLE_HANDLER)
+
+PATH_TO_COLORTABLES = os.path.join(os.path.dirname(plot_path), "rgb")
+
+TEMPDIR = "/tmp/color_maps"
+
+NCL_SCRIPT = """
+load "$NCARG_ROOT/lib/ncarg/nclscripts/csm/gsn_code.ncl"
+
+begin
+  print("Autogenerated NCL script is running.")
+  wks = gsn_open_wks("pdf","{{ outdir }}/available_colormaps_for_ncl")
+  {% for n in list_of_snippets %}{{n}} {% endfor %}
+end
+"""
+
+COLOR_SNIPPET = """
+  cmap := read_colormap_file("{{ path }}")
+  opt = True
+  opt@Frame = False
+  draw_color_palette(wks, cmap, opt)
+  gsn_text_ndc(wks, "{{ name }}", 0.5, 0.9, True)
+  frame(wks)
+"""
+
+
+def load_ncl_color_map(name):
+    """Load ncl color map to a list that is returned."""
+    def _is_first_character(full_string, single_character):
+        full_string = str(full_string).strip()
+        if full_string == "":
+            return False
+        return single_character == full_string[0]
+
+    def _format(content):
+        out = []
+        for item in content.split("\n"):
+            if 'ncolors' in item:
```

Comment:
A bit more readable would be:
```suggestion
            item = item.strip()
            if item and not ('ncolors' in item or item.startswith('#') or item.startswith(';')):
                out.append([int(elem) / 256
                            for elem in item.split()[0:3]] + [1])
```

Reply:
applied your suggestions in 
1f4b2aa